src/Tester/FSharpGrainTests.cs

```diff
@@ -14,20 +12,9 @@ namespace UnitTests.General
     /// Unit tests for grains implementing generic interfaces
     /// </summary>
     [TestClass]
-    public class FSharpGrainTests : UnitTestSiloHost
+    public class FSharpGrainTests : UnitTestSiloHostEnsureDefaultStartedSecondaryOff
```

Comment:
I guess a higher level question is whether these test cases should even be using a special case config of single silo, rather than using the same "normal" test config that all other tests uses?